rainbow-me/browser-extension
# Add `Rootstock` network

### Description
This PR adds `rootstock` network to the rainbow wallet. Rootstock is bitcoin sidechain with full compatibility with ethereum.

Website: https://rootstock.io
Defillama: https://defillama.com/chain/Rootstock
Explorer: https://explorer.rootstock.io
Developer portal: https://dev.rootstock.io 

### Demo
<img width="359" alt="Screenshot 2024-04-18 at 7 02 36 PM" src="https://github.com/rainbow-me/browser-extension/assets/104683677/65cbfc6d-c266-435b-b918-24751f023795">
<img width="356" alt="Screenshot 2024-04-18 at 7 04 17 PM" src="https://github.com/rainbow-me/browser-extension/assets/104683677/271d7ecc-d1f9-4e0b-bc51-0156cffb6171">
<img width="353" alt="Screenshot 2024-04-18 at 7 05 03 PM" src="https://github.com/rainbow-me/browser-extension/assets/104683677/7b9ec5cb-451f-40c2-85ba-4124d367624d">
<img width="352" alt="Screenshot 2024-04-18 at 7 04 48 PM" src="https://github.com/rainbow-me/browser-extension/assets/104683677/5f3510c5-f319-4525-960a-c1b3ecaca969">
<img width="352" alt="Screenshot 2024-04-18 at 7 03 32 PM" src="https://github.com/rainbow-me/browser-extension/assets/104683677/9e61bced-f88f-4d6b-b2cf-3e68821bb7b0">
<img width="352" alt="Screenshot 2024-04-18 at 7 02 51 PM" src="https://github.com/rainbow-me/browser-extension/assets/104683677/760d2575-68ed-4bd6-b7b1-ce13383f2db0">


### Testing
- Rootstock network
- Send / Receive RBTC
- See balance of RBTC and ERC20 Tokens
